SOURCEBUILD.md

```diff
@@ -144,6 +144,26 @@ This project uses cmake for building the C/C++ library. To configure cmake, run:
   ```
 The default installation locations are `$CMAKE_INSTALL_PREFIX/lib` and `$CMAKE_INSTALL_PREFIX/include/cugraph` respectively.
 
+#### Updating the RAFT branch
+
+`libcugraph` uses the [RAFT](https://github.com/rapidsai/raft) library and there are times when it might be desirable to build against a different RAFT branch, such as when working on new features that might span both RAFT and cuGraph. To change the RAFT branch used to build `libcugraph`, modify the `cpp/cmake/thirdparty/get_raft.cmake` file as follows:
+
+```cmake
+# Change pinned tag and fork here to test a commit in CI
+# To use a different RAFT locally, set the CMake variable
+# RPM_raft_SOURCE=/path/to/local/raft
+find_and_configure_raft(VERSION    ${CUGRAPH_MIN_VERSION_raft}
+                        FORK       <your_git_fork>
+                        PINNED_TAG <your_git_branch_or_tag>
+
+                        # When PINNED_TAG above doesn't match cugraph,
+                        # force local raft clone in build directory
+                        # even if it's already installed.
+                        CLONE_ON_PIN     ON
+                        )
+```
+
+If the above change is pushed to a pull request, the continuous integration servers will use the specified RAFT branch to run the cuGraph tests. For local development, the `CPM_raft_SOURCE=<path/to/raft/source>` option can be passed to the `cmake` command to enable `libcugraph` to use the local RAFT branch instead.
```

Comment:
Thanks for this often overlooked update to the docs!

> For local development, the `CPM_raft_SOURCE=<path/to/raft/source>` option can be passed to the `cmake` command to enable `libcugraph` to use the local RAFT branch instead.

I wonder if this should be mentioned first, before the example showing how to change `get_raft.cmake`, since (I hope) just passing `CPM_raft_SOURCE` is the more common use case for changing the version of RAFT being used.
And I know this is somewhat vague, but can you also elaborate on when a user should need to change `get_raft.cmake` vs. just passing in the cmake option? The descriptions for both currently sound very similar ("working on new features" vs. "local development")